README.md

```diff
@@ -1,52 +1,101 @@
+
+**This mod cannot be installed on Android/iOS/Epic Games/Microsoft Store**
 
 # What does the mod add?
 
 ## Medic
+### **Team: Crewmates**
 The Medic can give any player a shield that will make them immortal. Although, if The Medic dies, the shield will break.
 The only exception is The Officer; they will still die if they try to kill a Crewmate.
 The Medic's other feature shows when they find a corpse: they can get a report that contains clues to the killer's identity.
 The type of information they get is based on a timer that can be configured inside the lobby.
+
+## Game Options:
+| Name | Description | Type | Default |
+|----------|:-------------:|:------:|:------:|
+| Snow Medic | This is the game setting that toggles whether the Medic's name is lit up green for everybody in the game, or just themselves | Toggle | Off |
+| Show Shielded Player | When The Medic shield's somebody, their visor will change to the color cyan. If this setting is set to 1, everybody can see the color change. If not, only the shielded player can see | Options | Self |
+| Murder Attempt Indicator For Shielded Player | If this setting is enabled, the shielded player will hear a *ting* noise when somebody tries (and fails) to murder them | Toggle | On |
+| Snow Medic Reports | When the medic reports the body and if this function is desactived, it will not show the Medic who reported it | Options | Off |
+| Time Where Medic Reports Will Have Name | The amount of time (in seconds) that The Medic will have to report the body since death to get the killer's name | Number | 5 |
+| Time Where Medic Reports Will Have Color Type | The amount of time (in seconds) that The Medic will have to report the body since death to get the killer's color type. "color type" means either "lighter" or "darker", and a full list of colors and their types are included at the bottom of the page | Number | 20 |
+| Medic Spawn Chance | The percentage chance that anybody in the game will become The Medic | Number | 100% |
+-----------------------
   
 ## Officer
+### **Team: Crewmates**
 The Officer is a class of Crewmate that is allowed to kill people, similar to Impostors.
 Their goal is to locate the Impostor and deliver vigilante justice, but if they accidentally shoot a Crewmate, they die instead.
+
+### Game Options
+| Name | Description | Type | Default |
+|----------|:-------------:|:------:|:------:|
+| Snow Officer | If this setting is enabled, The Officer's name will be lit up blue for everyone. If it isn't, it will only be lit for themselves | Toggle | Off |
+| Officer Kill Cooldown | This is the kill cooldown length for The Officer. The first cooldown on the first round will be equal to ten no matter what, just like the Impostor | Number | 30 |
+| Officer Kill Behaviour | This settings control what will happen when The Officer attempts to kill someone in the game | Options | Impostor |
```

Comment:
I just took the descriptions that NotHunter101 put in, but I’m probably going to redo them and make them a lot better than what he put in, I won’t do it now and I’ll probably do it on Monday because it would take a while and also, I want to rest one on the weekend 😎 but if you want to edit and do it for me (if you want) I would appreciate it very much